src/settings/LoanHistory/LoanHistoryForm.js

```diff
@@ -54,7 +56,7 @@ class LoanHistoryForm extends Component {
             data-test-loan-history-save-button
             type="submit"
             buttonStyle="primary paneHeaderNewButton"
-            disabled={pristine || submitting}
+            disabled={pristine || submitting || !stripes?.user?.perms?.['ui-circulation.settings.edit-loan-history']}
```

Comment:
I'm not sure that we need optional chaining here. Perms object should be available on stripes all the time. If not, please ignore this comment.

Reply:
Yes, I do agree that Perms object will be there. But I think it is always safe to ensure optional chaining!!